src/transformers/models/llama/modeling_llama.py

```diff
@@ -654,7 +654,7 @@ def forward(
             position_ids = torch.arange(
                 past_key_values_length, seq_length + past_key_values_length, dtype=torch.long, device=device
             )
-            position_ids = position_ids.unsqueeze(0).view(-1, seq_length)
+            position_ids = position_ids.unsqueeze(0)
         else:
             position_ids = position_ids.view(-1, seq_length).long()
```

Comment:
Isn't this one a noop as well when using transformers `generate`?

Reply:
Which one?

Reply:
yeah agreed!

Reply:
Missed this. I'll address it soon

Reply:
Done, I've removed all the instances that match this pattern